src/mmw/sass/_bootstrap.scss

```diff
@@ -5,12 +5,12 @@
 // Reset and dependencies
 @import "bootstrap/normalize";
 @import "bootstrap/print";
-@import "bootstrap/glyphicons";
+// @import "bootstrap/glyphicons";
```

Comment:
We're not using some of those dependencies now, but we might want to use
them later. Commenting them out would make it easier to find and activate
them later. But I'm not opposed to completely removing them.

On Fri, Sep 25, 2015 at 5:18 PM, lliss <notifications@github.com> wrote:

> In src/mmw/sass/_bootstrap.scss
> <https://github.com/WikiWatershed/model-my-watershed/pull/831#discussion_r40476561>
> :
>
> > @@ -5,12 +5,12 @@
> >  // Reset and dependencies
> >  @import "bootstrap/normalize";
> >  @import "bootstrap/print";
> > -@import "bootstrap/glyphicons";
> > +// @import "bootstrap/glyphicons";
>
> Is there any harm to just deleting these?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/WikiWatershed/model-my-watershed/pull/831/files#r40476561>
> .
>



-- 
Jeff Frankl, UI/UX Designer

Azavea  |  340 N 12th St, Suite 402, Philadelphia, PA
Web azavea.com  |  Blog azavea.com/blogs  |  Twitter @azavea
<http://twitter.com>